{"detail_link": "https://huggingface.co/datasets/HuggingFaceH4/grok-conversation-harmless", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/huggingface.py\", line 399, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/huggingface.py\", line 412, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x00000001049aae00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x00000001049a2d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001044ed0b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001045345d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104575980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x00000001045288f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x000000010496e478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001049716a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x000000010494f3b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000104971f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001049404a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000104991644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001049917d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x00000001049a294c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://huggingface.co/datasets/allenai/WildChat-4.8M-Full", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/huggingface.py\", line 399, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/huggingface.py\", line 412, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x00000001050a2e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010509ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104be50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104c2c5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c6d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104c208f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105066478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050696a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050473b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105069f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001050384a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105089644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050897d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010509a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://huggingface.co/datasets/allenai/WildChat-1M-Full", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/huggingface.py\", line 399, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/huggingface.py\", line 412, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x00000001050a2e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010509ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104be50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104c2c5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c6d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104c208f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105066478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050696a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050473b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105069f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001050384a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105089644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050897d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010509a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://huggingface.co/datasets/allenai/ValuePrism", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/huggingface.py\", line 399, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/huggingface.py\", line 412, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x00000001050a2e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010509ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104be50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104c2c5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c6d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104c208f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105066478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050696a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050473b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105069f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001050384a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105089644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050897d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010509a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_finance_economics/5", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Automobiles/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_computer_communication/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_technology_scientific_research/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_petrochemical/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_subject_education_education/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_tourism_geography/4", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_real_estate_construction/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_biomedicine/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_transportation/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_law_judiciary/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_medicine_health_psychology_traditional_chinese_medicine/11", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_literature_emotion/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_mathematics_statistics/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_water_resources_ocean/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_film_entertainment/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_agriculture_forestry_animal_husbandry_fishery/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_sports/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_current_affairs_government_administration/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_automobile/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_other_manufacturing/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_news_media/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_electric_power_energy/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/OpenSeek-Pretrain-100B-jsonl/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_fire_safety_food_safety/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_mining/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_other_information_services_information_security/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_game/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_artificial_intelligence_machine_learning/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_computer_programming_code/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_accommodation_catering_hotel/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryCorpus2_aerospace/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Travel-Geography/4", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Finance-Economics/5", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Transportation/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Health-Medicine/40", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Subject-Education/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Technology-Research/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Hospitality-Catering/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Aerospace/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Literature-Emotions/1", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Artificial-Intelligence/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/BAAI/IndustryInstruction_Law-Justice/2", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000105062e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010505ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000104ba50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000104bec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000104c2d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000104be08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000105026478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001050296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001050073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000105029f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x0000000104ff84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000105049644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001050497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010505a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/OpenGVLab/VRBench/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x000000010450ee00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x0000000104506d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001040510b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001040985d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x00000001040d9980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x000000010408c8f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001044d2478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001044d56a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001044b33b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001044d5f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001044a44a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001044f5644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001044f57d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010450694c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/OpenGVLab/InternVL-eval/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x000000010450ee00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x0000000104506d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001040510b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001040985d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x00000001040d9980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x000000010408c8f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001044d2478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001044d56a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001044b33b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001044d5f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001044a44a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001044f5644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001044f57d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010450694c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/OpenGVLab/VideoMAEv2-TAL-Features/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x000000010450ee00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x0000000104506d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001040510b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001040985d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x00000001040d9980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x000000010408c8f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001044d2478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001044d56a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001044b33b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001044d5f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001044a44a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001044f5644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001044f57d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010450694c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/OpenGVLab/LongVid/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x000000010450ee00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x0000000104506d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001040510b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001040985d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x00000001040d9980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x000000010408c8f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001044d2478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001044d56a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001044b33b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001044d5f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001044a44a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001044f5644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001044f57d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010450694c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/OpenGVLab/NIAH-Video/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x000000010450ee00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x0000000104506d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001040510b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001040985d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x00000001040d9980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x000000010408c8f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001044d2478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001044d56a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001044b33b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001044d5f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001044a44a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001044f5644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001044f57d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010450694c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/OpenGVLab/ScaleVLN/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x000000010450ee00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x0000000104506d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001040510b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001040985d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x00000001040d9980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x000000010408c8f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001044d2478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001044d56a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001044b33b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001044d5f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001044a44a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001044f5644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001044f57d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010450694c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/OpenGVLab/InternVL-Domain-Adaptation-Data/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x000000010450ee00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x0000000104506d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001040510b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001040985d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x00000001040d9980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x000000010408c8f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001044d2478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001044d56a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001044b33b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001044d5f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001044a44a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001044f5644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001044f57d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010450694c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/OpenGVLab/MMT-Bench/2", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x000000010450ee00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x0000000104506d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001040510b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001040985d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x00000001040d9980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x000000010408c8f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001044d2478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001044d56a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001044b33b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001044d5f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001044a44a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001044f5644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001044f57d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010450694c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/Shanghai_AI_Laboratory/SWE-Fixer-Eval/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000102a0ae00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x0000000102a02d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x000000010254d0b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x00000001025945d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x00000001025d5980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x00000001025888f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001029ce478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001029d16a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001029af3b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001029d1f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001029a04a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001029f1644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001029f17d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x0000000102a0294c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/iic/frames/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000100882e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010087ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001003c50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x000000010040c5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x000000010044d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x00000001004008f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000100846478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001008496a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001008273b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000100849f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001008184a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000100869644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001008697d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010087a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/iic/docmath/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000100882e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010087ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001003c50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x000000010040c5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x000000010044d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x00000001004008f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000100846478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001008496a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001008273b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000100849f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001008184a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000100869644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001008697d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010087a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/iic/longbench/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000100882e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010087ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x00000001003c50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x000000010040c5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x000000010044d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x00000001004008f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000100846478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001008496a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001008273b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000100849f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001008184a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000100869644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001008697d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010087a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://www.modelscope.cn/datasets/OpenBMB/llava_zh/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000101462e00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x000000010145ad00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000100fa50b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000100fec5d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x000000010102d980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000100fe08f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x0000000101426478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001014296a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x00000001014073b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x0000000101429f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001013f84a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x0000000101449644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001014497d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x000000010145a94c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
{"detail_link": "https://modelscope.cn/datasets/meituan/CoreCodeBench-Source_Copy/0", "category": "datasets", "error_msg": "Traceback (most recent call last):\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 338, in scrape\n    metadata = self.get_dataset_info()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/libs/oslm-crawler/src/oslm_crawler/crawler/modelscope.py\", line 354, in get_dataset_info\n    WebDriverWait(self.driver, 5).until(\n  File \"/Users/liaofeng/Documents/Codespace/InsightSwarm/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x00000001030eae00 cxxbridge1$str$ptr + 2742224\n1   chromedriver                        0x00000001030e2d00 cxxbridge1$str$ptr + 2709200\n2   chromedriver                        0x0000000102c2d0b8 cxxbridge1$string$len + 90520\n3   chromedriver                        0x0000000102c745d8 cxxbridge1$string$len + 382648\n4   chromedriver                        0x0000000102cb5980 cxxbridge1$string$len + 649824\n5   chromedriver                        0x0000000102c688f4 cxxbridge1$string$len + 334292\n6   chromedriver                        0x00000001030ae478 cxxbridge1$str$ptr + 2494024\n7   chromedriver                        0x00000001030b16a4 cxxbridge1$str$ptr + 2506868\n8   chromedriver                        0x000000010308f3b0 cxxbridge1$str$ptr + 2366848\n9   chromedriver                        0x00000001030b1f4c cxxbridge1$str$ptr + 2509084\n10  chromedriver                        0x00000001030804a8 cxxbridge1$str$ptr + 2305656\n11  chromedriver                        0x00000001030d1644 cxxbridge1$str$ptr + 2637844\n12  chromedriver                        0x00000001030d17d0 cxxbridge1$str$ptr + 2638240\n13  chromedriver                        0x00000001030e294c cxxbridge1$str$ptr + 2708252\n14  libsystem_pthread.dylib             0x000000019e82fc0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000019e82ab80 thread_start + 8\n\n"}
